requirements.txt

```diff
@@ -10,6 +10,7 @@ matrix-nio[e2e] >= 0.15.0, < 0.16
 mistune         >= 0.8.4,  < 0.9
 pymediainfo     >= 4.2.1,  < 5
 plyer           >= 1.4.3,  < 2
+dbus-python
```

Comment:
This should be
```

dbus-python     >= 1.2.16, < 2; platform_system == "Linux"

```

Reply:
The corresponding change made in the separate commit